analyzers/src/SonarAnalyzer.CSharp/Rules/CognitiveComplexity.cs

```diff
@@ -90,6 +90,16 @@ protected override void Initialize(ParameterLoadingAnalysisContext context)
                c => CheckComplexity<FieldDeclarationSyntax>(c, f => f, f => f.Declaration.Variables[0].Identifier.GetLocation(),
                    CSharpCognitiveComplexityMetric.GetComplexity, "field", Threshold),
                SyntaxKind.FieldDeclaration);
+
+            context.RegisterSyntaxNodeActionInNonGenerated(c =>
+            {
+                if (((LocalFunctionStatementSyntaxWrapper)c.Node).Modifiers.Any(SyntaxKind.StaticKeyword))
+                {
+                   CheckComplexity<SyntaxNode>(c, m => m, m => ((LocalFunctionStatementSyntaxWrapper)m).Identifier.GetLocation(),
+                       CSharpCognitiveComplexityMetric.GetComplexity, "static local function", Threshold);
```

Comment:
pedantic: I think we should either put all parameters on the same line or all of them on separate lines.